support/testing/cid.go

```diff
@@ -8,10 +8,25 @@ import (
 var DefaultHashFunction = uint64(mh.BLAKE2B_MIN + 31)
 var DefaultCidBuilder = cid.V1Builder{Codec: cid.DagCBOR, MhType: DefaultHashFunction}
 
-func MakeCID(input string) cid.Cid {
+func MakeCID(input string, prefix *cid.Prefix) cid.Cid {
```

Comment:
We _really_ should be using `cid.Builder`, not `*cid.Prefix`.

Reply:
@Stebalien could you please follow up to implement a better pattern here so that future work doesn't propagate this further?

Reply:
After trying to fix this, I get what it's doing. We can't use a cid.Builder because this is making a fake CID.